app/src/main/java/com/android/unio/ui/event/EventDetails.kt

```diff
@@ -156,22 +163,27 @@ fun EventScreenScaffold(
     eventViewModel: EventViewModel,
     userViewModel: UserViewModel
 ) {
+  val nbOfTabs = 2
```

Comment:
IMO this constant should be stored outside the composable, e.g. in a companion object in the same file for example.